src/core/Table/cells/DefaultCell.tsx

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { CellRendererProps } from 'react-table';
+
+export type DefaultCellProps<
+  T extends Record<string, unknown>
+> = CellRendererProps<T> & React.ComponentPropsWithoutRef<'div'>;
+
+/**
+ * Default cell.
+ * It should be passed to `cellRenderer`.
+ * In most cases you shouldn't need to pass this component.
```

Comment:
Added this so users could add some native attributes on the Cell itself, right? So maybe instead of this comment, we could say that `Use this component to add native html attributes to the cell container` or something like that...

Reply:
Changed.